### DISCUSSION TITLE
终端全屏时强制关闭网页后打开显示出错

### DISCUSSION THREAD

**Discussion by NameMicrobe7931** [1Panel-dev/1Panel#6268] [Q&A]
### 联系方式

653838873@qq.com

### 1Panel 版本

v1.10.13-lts

### 问题描述

主机-终端内点击旁边的全屏后按esc键退出全屏，重新打开页面后显示错误，旁边的快捷按钮和快捷栏消失，主机架构为armv7,多次尝试重现均能实现，概率接近百分之百，只能重新安装，但重新安装会导致面板的所有配置丢失，可能其他架构的主机也会出现同样问题
![屏幕截图(111)](https://github.com/user-attachments/assets/5ab26ec0-a227-46b7-882c-f0d8e040115c)
![屏幕截图(110)](https://github.com/user-attachments/assets/d06dfa9a-62e1-4cd6-bb95-6810f708531c)
![屏幕截图(112)](https://github.com/user-attachments/assets/6af9a2f3-d6e7-4ebe-a99d-3e0298594a7f)

### 重现步骤

1. armv7主机安装后打开面板，进入主机-终端内连接本机
2. 点击终端右上角的全屏按钮，edge提示“按esc键退出全屏”，按下后网页没有正常出现左侧的快捷栏
3. 直接关闭页面，重新打开面板登录

### 期待的正确结果

点击全屏按钮，按下esc键后正常显示旁边的快捷栏

**Comment by NameMicrobe7931** [comment]
ssh内重新启动也解决不了

**Reply by wanghe-fit2cloud** [reply]
目前的全屏效果是覆盖整个显示器，如果将其调整为仅在浏览器内全屏，是否能满足您的需求？

**Reply by NameMicrobe7931** [reply]
只要求面板全屏后关闭网页重新打开不把旁边的快捷栏卡不见就行，另外更新的时候建议在1pctl内加入更新面板命令，此外再推一份快速更新的sh脚本，我两台服务器都把下方的更新按钮卡消失了，不能更新

**Reply by wanghe-fit2cloud** [reply]
其他的需求建议新建 issue，方便后续跟踪。
